playbooks/cluster/openshift/config.yml

```diff
@@ -84,14 +77,6 @@
 
 - import_playbook: "{{ openshift_ansible_dir }}/playbooks/prerequisites.yml"
 
-- hosts: masters nodes
```

Comment:
you can pre-pull images too for cri-o, just use `podman` instead of `docker`

Reply:
Yes, but we install using rpms (`containerized=fase`)

Reply:
I think it still downloads some images from docker hub, worth to check.